.github/ISSUE_TEMPLATE/other_issue.yaml

```diff
@@ -0,0 +1,40 @@
+name: Other Issue
+description: Questions and issues that do not fit in enlisted types.
+title: "[Bug]: "
+labels: [bug]
+body:
+  - type: textarea
+    id: describe-bug
+    attributes:
+      label: Describe the bug
+      description: Describe what happens and the steps to recreate the problem. Issues that we can't reproduced will be closed.
+    validations:
+      required: true
+  - type: textarea
+    id: screenshots
+    attributes:
+      label: Video/Screenshots
+      description: Please provide a screenshot or a short video presenting the issue
+    validations:
+      required: true
+  - type: input
+    id: demo
+    attributes:
+      label: Provide a link to the demo with the bug reproduction
+      description: Use StackBlitz, CodeSandbox, CodePen, JSFiddle, or any other online sandbox.
+  - type: input
+    id: framework
+    attributes:
+      label: Framework version
+  - type: input
+    id: environment
+    attributes:
+      label: Your environment
+      description: What browser and operating system are you using?
+    validations:
+      required: true
```

Comment:
I need a template with a few inputs to create tasks quickly.

Besides, these inputs don't really make sense for issues other than bugs.

```suggestion
```